stock_location_tank/views/stock_location_views.xml

```diff
@@ -0,0 +1,98 @@
+                        />
+                        <label
+                            for="capacity"
+                            string="Capacity"
+                            attrs="{'invisible': ['|', ('is_tank', '=', False), ('usage', '=', 'view')]}"
+                        />
+                        <div class="o_row no-gutters d-flex">
+                            <field
+                                name="capacity"
+                                class="oe_inline text-left"
+                                attrs="{'invisible': ['|', ('is_tank', '=', False), ('usage', '=', 'view')], 'required': [('is_tank', '=', True), ('usage', '!=', 'view')]}"
+                            />
+                            <field
+                                name="allowed_uom_id"
+                                nolabel="1"
+                                attrs="{'invisible': ['|', ('is_tank', '=', False), ('usage', '=', 'view')], 'required': [('is_tank', '=', True), ('usage', '!=', 'view')]}"
+                                class="oe_inline"
+                            />
+                            <field
+                                name="percentage_occupied"
+                                widget="percentpie"
+                                nolabel="1"
+                                string=" "
+                                attrs="{'invisible': ['|', ('is_tank', '=', False), ('usage', '=', 'view')]}"
+                            />
+                        </div>
+                        <field
+                            name="sequence_id"
+                            attrs="{'invisible': ['|', ('is_tank', '=', False), ('usage', '=', 'view')], 'required': [('is_tank', '=', True), ('usage', '!=', 'view')]}"
+                        />
+                        <field
+                            name="allowed_product_ids"
+                            widget="many2many_tags"
+                            attrs="{'invisible': ['|', ('is_tank', '=', False), ('usage', '=', 'view')], 'required': [('is_tank', '=', True), ('usage', '!=', 'view')]}"
+                        />
+                    </group>
+                </group>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="view_location_tree2" model="ir.ui.view">
+        <field name="name">stock.location.tree.inherit</field>
+        <field name="model">stock.location</field>
+        <field name="inherit_id" ref="stock.view_location_tree2" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='complete_name']" position="before">
+                <field name="is_blocked" invisible="1" />
+                <field
+                    string=" "
+                    name="json_popover"
+                    widget="stock_rescheduling_popover"
+                    attrs="{'invisible': [('is_blocked', '=', False)]}"
+                    width="0.1"
```

Comment:
When you use this hard-coded values, ensure that everything looks right in mobile view